packages/env/lib/config.js

```diff
@@ -293,9 +293,25 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 		};
 	}
 
+	const wpOrgFields = sourceString.match(
+		/^https?:\/\/downloads\.wordpress\.org\/(?:plugin|theme)\/([^\s\.]*)([^\s]*)?\.zip$/
+	);
+	if ( wpOrgFields ) {
+		return {
+			type: 'zip',
+			url: sourceString,
+			path: path.resolve(
+				workDirectoryPath,
+				encodeURIComponent( wpOrgFields[ 1 ] )
+			),
+			basename: encodeURIComponent( wpOrgFields[ 1 ] ),
+		};
+	}
+
```

Comment:
`parseSourceString` has unit tests. These should be updated to test this new functionality.

Reply:
added unit tests.